src/derivatives/atlas.md

```diff
@@ -8,45 +8,43 @@ segmentations, and other knowledge annotations such as landmarks defined with
 respect to individual or template brains that are supported by spaces
 such as surfaces and regular grids (images).
 
-For derivatives with atlases in their provenance corresponding to individual subjects,
-the organization follows the standards for BIDS raw and derivatives.
+The organization follows the standards for BIDS raw and derivatives.
 The following entities MAY be employed to specify template- and atlas-derived results:
 
 -    [`tpl-<label>`](../glossary.md#template-entities) is REQUIRED to specify derivatives defining
-     a [template](../common-principles.md).
+     a [template](../common-principles.md).  MUST NOT be used alongside `atlas-<label>`
+-    [`atlas-<label>`](../glossary.md#atlas-entities) is REQUIRED to specify derivatives defining an [atlas](../common-principles.md).  MUST NOT be used alongside `template-<label>`
```

Comment:
Thanks for these changes, @oesteban, and let's have a meeting soon to discuss this further. @pwighton - please correct me if I am wrong, but I think @pwighton is arguing that an atlas is currently defined as being a segmentation (defined on a template), while tpl is a an averaged feature map (i.e. template) across subjects. Therefore, it may seem unintuitive to have both tpl- and atlas- in the filename, as it cannot both be a feature map and a segmentation? The space entity would reconcile how atlas- can be used without tpl-. But if tpl- serves the same role as space- then having tpl- and atlas- in the name still makes sense, but is then an atlas and not a template.

Reply:
Yes, that was the understanding I arrived at later (https://github.com/oesteban/bids-specification/pull/1#discussion_r1879828149).

As I argue there, the data type (whether an average feature map or a segmentation) in BIDS is expressed by the suffix. This use of atlas- is just reflecting some suffixes, and leaving tpl- for others, which is counter-productive because we have suffix that fully expresses this (and conflicting with BIDS, which doesn't change sub- depending on the suffix).

I think the best way to approach this is through the homology/analogy between subject and template and between session and cohort. If we cannot agree on that mapping, hardly we're going to find a common place for atlas.

Reply:
> I think the best way to approach this is through the homology/analogy between subject and template and between session and cohort. If we cannot agree on that mapping, hardly we're going to find a common place for atlas.

I've raised this point previously, but dropped it because it didn't seem critical, but it looks like it will be.  Cohort in the biological sciences generally means "a set of subjects who satisfy a set of inclusion criteria" and while this can be used to denote a session or timepoint, e.g. "subjects who have been imaged between Jan and Dec 2024",  it can be used for other things as well, e.g "Subjects who were born before 1980" or "Subjects who have a BMI less than 20" or "Subjects who speak Italian".

Here are how some other controlled terminologies define cohort:

"In OHDSI research, we define a cohort as a set of persons who satisfy one or more inclusion criteria for a duration of time." [1]

"A group of individuals who share a common exposure, experience or characteristic or a group of individuals followed-up or traced over time in a cohort study. [AMA Manual of Style]" [2]

"Cohort Study: Observational study that assembles a large group of individuals, collects extensive information about
these people at baseline, and follows up with the group for a period of time in an attempt to examine
relationships between outcomes and risk factors." [3]

[1]: https://ohdsi.github.io/TheBookOfOhdsi/Cohorts.html
[2]: https://evs.nci.nih.gov/ftp1/CDISC/Glossary/CDISC%20Glossary.html
[3]: https://ismpp.memberclicks.net/assets/docs/Committees/updated%20glossary_final_0716.pdf

Reply:
Sure, the homology is scoped within BIDS:

tpl- works like sub- (and also provides a reference frame for the results within the folder tpl-<label>/ like sub-<label>/)
cohort- works like session- (and also provides a grouping mechanism within the corresponding folder)

beyond that, this mapping does not intend to make cohorts an equivalent to sessions.